routes/users.js

```diff
@@ -175,47 +163,42 @@ router.get('/agency', redirectLogin, async (req, res) => {
       user: res.locals.user,
     });
   } catch (err) {
-    return sendError(res, 400, err);
+    return handleError(res, 400, err);
   }
 });
 
 // @desc    agency info is sent to db
 // @route   POST '/users/agency'
 // @access  private, partners only
 // @tested 	No
-router.post(
-  '/agency',
-  createAgencyValidationRules(),
-  validate,
-  async (req, res) => {
-    const { agencyName, agencyWebsite, agencyPhone, agencyBio } = req.body;
-
-    const newAgency = new Agency({
-      agencyName,
-      agencyWebsite,
-      agencyPhone,
-      agencyBio,
-      accountManager: req.session.user._id,
-    });
-    try {
-      await newAgency.save();
+router.post('/agency', createAgencyValidationRules(), validate, async (req, res) => {
+  const { agencyName, agencyWebsite, agencyPhone, agencyBio } = req.body;
+
+  const newAgency = new Agency({
+    agencyName,
+    agencyWebsite,
+    agencyPhone,
+    agencyBio,
+    accountManager: req.session.user._id,
+  });
+  try {
+    await newAgency.save();
 
-      return res.status(200).send({
-        success: true,
-        user: req.session.user,
-        url: '/users/profile',
-      });
-    } catch (err) {
-      console.log(err);
-      return sendError(res, 400, err);
-    }
+    return res.status(200).send({
+      success: true,
+      user: req.session.user,
+      url: '/users/profile',
+    });
+  } catch (err) {
+    console.log(err);
+    return handleError(res, 400, err);
   }
-);
+});
 
 const sendEmail = (email, verificationHash) => {
   sendVerificationEmail(email, verificationHash).then((emailResponse) => {
```

Comment:
we decided to go with async/await for the project

Reply:
not sure if thats already existing code

Reply:
can you reformat this into async/await?

Reply:
that's exisiting code, I only added the new errorHandler but can refactor it if you wish

Reply:
yeah it's easier if you just reformat it quickly, thank you